pkg/config/defaults.go

```diff
@@ -87,17 +86,17 @@ func DefaultResource(name string, terraformSchema *schema.Resource, opts ...Reso
 	// - aws_rds_cluster => Cluster
 	// - aws_rds_cluster_parameter_group => ClusterParameterGroup
 	// - kafka_topic => Topic
-	kind := strcase.ToCamel(strings.Join(words[2:], "_"))
+	kind := tjname.NewFromSnake(strings.Join(words[2:], "_")).Camel
```

Comment:
Otherwise, `strcase.ToCamel` treates `ec2` as two words `Ec` and `2`, which causes it not to be found in the acronyms list.

Reply:
I needed this in provider-jet-gcp as well for hl7 to HL7 👍